### PR TITLE
Bump/recipes

### PR DESCRIPTION
**Pull request by texierp**

### CODE REVIEW

File: meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb [hunk 1]
```diff
@@ -10,12 +10,15 @@ DEPENDS = " \
     virtual/libiconv \
 "
 
+PV = "3-6-0+git${SRCPV}"
```
Comment (kraj) [inline meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb]: this should be  `3.6.0+git${SRCPV}`

File: meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 inherit setuptools3
 require python-lxml.inc
 CLEANBROKEN = "1"
+
```
Comment (kraj) [inline meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb]: this is redundant empty line

File: meta-python/recipes-devtools/python/python3-sqlparse_0.3.1.bb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 inherit setuptools3
 require python-sqlparse.inc
 BBCLASSEXTEND = "native nativesdk"
+
```
Comment (kraj) [inline meta-python/recipes-devtools/python/python3-sqlparse_0.3.1.bb]: remove empty line

Comment (texierp) [reply meta-python/recipes-devtools/python/python3-sqlparse_0.3.1.bb]: same as above

File: meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb (hunk 2 above)
Comment (texierp) [reply meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb]: this empty line was added by the `upgradehelper.py` script, should I delete it anyway?

File: meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb (hunk 1 above)
Comment (texierp) [reply meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb]: In fact, `3-6-0` is the release tag from lm-sensors [1], with 3.6.0, I raised the following issue:

```
INFO: Skip package lmsensors (status = UNKNOWN_BROKEN, current version = 3.6.0, next version = 3-6-0)
```

[1]  https://github.com/lm-sensors/lm-sensors/releases

Comment (kraj) [reply meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb]: what happens when you set `PE = "1" ` instead of setting PV ?

Comment (texierp) [reply meta-oe/recipes-bsp/lm_sensors/lmsensors_3.6.0.bb]: Same, but anyway, I will drop this patch from this series ;).
I will investigate more, and I'll send another PR.

File: meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb (hunk 2 above)
Comment (texierp) [reply meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb]: fixed

**Comment by texierp** [comment]
Should be fine now @kraj ;)

**Comment by texierp** [comment]
Now in master

**Pull request closed without merging**